packages/eslint-plugin/src/rules/max-params.ts

```diff
@@ -0,0 +1,91 @@
+import { AST_NODE_TYPES, type TSESTree } from '@typescript-eslint/utils';
+
+import * as util from '../util';
```

Comment:
Nit: In the meantime, we switched the style to named imports (#7669). I can make the change here since it's good to merge other than these nits.